### PR TITLE
Add a small discussion for longer ranking tables

### PR DESCRIPTION
**Pull request by drvinceknight**

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -568,6 +569,11 @@ \subsection{The special case of \(N=2\)}\label{sec:two_individuals}
         Section~\ref{sec:strong_invaders}.
 \end{enumerate}
 
+It is also noted that TF1, TF2 and TF3 all perform well, other finite state
```
Comment (marcharper) [inline main.tex]: Prober 3 and 4 are handshakers... I guess they are technically finite state machines. Maybe you were thinking of Fortress 3 and 4 (which are handshakers).

File: main.tex [hunk 2]
```diff
@@ -696,15 +702,16 @@ \subsection{Strong invaders}\label{sec:strong_invaders}
...
+Only one of the above strategies is stochastic although close inspection of the
 source code of PSO Gambler shows that it makes stochastic decisions rarely, and
-is functionally very similar to its deterministic cousin Evolved Looker Up. Apart
-from TF3 in \(N=3\), the finite state machines trained specifically for Moran
-processes do not appear in the top 5: whereas strategies trained for tournaments do.
-This is due to the nature of invasion: most of the opponents will initially be different
-strategies. The next section will consider the converse situation.
+is functionally very similar to its deterministic cousin Evolved Looker Up.
+The PSO Gambler Mem1 strategy is a memory one strategy that has been trained to
+maximise it's utility and does perform well.
```
Comment (marcharper) [inline main.tex]: its

File: main.tex [hunk 3]
```diff
@@ -696,15 +702,16 @@ \subsection{Strong invaders}\label{sec:strong_invaders}
...
+Apart from TF3, the finite state machines trained specifically for
+Moran processes do not appear in the top 5: whereas strategies trained for
```
Comment (marcharper) [inline main.tex]: ':' -> ',', whereas -> while

Comment (drvinceknight) [reply main.tex]: Dammit... My attempt to use fancy punctuation and words foiled once again...

**Comment by drvinceknight** [comment]
Have made those changes @marcharper.

**Pull request merged**